mvpa2/clfs/gda.py

```diff
@@ -259,3 +259,94 @@ def _g_k(self, data):
             dm = data - m
             res.append(b - 0.5 * np.sum(np.dot(dm, covi) * dm, axis=1))
         return np.array(res).T
+
+class ShrinkageLDA(GDA):
+    """LDA with Ledoit-Wolf estimate of covariance matrix
+    """
+
+    __tags__ = GDA.__tags__ + ['linear', 'lda', 'shrinkage']
+
+    def __init__(self, shrinkage_estimator='LedoitWolf', block_size=1000, \
+    **kwargs):
```

Comment:
shave a look at some other classifiers, e.g. at GDA or others to how to ideally define parameters (via those `Parameter` definitions in the class)

no need for trailing `\` here since it is within `()`